libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -272,7 +273,7 @@ setTTL ttl (WithStatusBase s ls c _) = WithStatusBase s ls c (pure ttl)
 setWsTTL :: FeatureTTL -> WithStatus cfg -> WithStatus cfg
 setWsTTL = setTTL
 
-type WithStatus (cfg :: Type) = WithStatusBase Identity cfg
+type WithStatus = WithStatusBase Identity
```

Comment:
i liked the unreduced variant better.

Reply:
I made this change because I wanted to use `WithStatus` as an argument of a higher-rank type constructor. I gave up with that approach, so this could be reverted, but I'm curious: why do you like the other one better? This is usable in more cases, and perfectly equivalent otherwise.

Reply:
I'll leave this in, if you don't mind too much. It's the normal way to define a type synonym.